chart/templates/service-monitor.yaml

```diff
@@ -67,7 +67,10 @@ spec:
       keySecret:
         name: vc-{{ .Release.Name }}
         key: client-key
-  {{- if .Values.controlPlane.advanced.virtualScheduler.enabled }}
+  {{- if or
+      (and .Values.controlPlane.distro.k8s.enabled .Values.controlPlane.distro.k8s.scheduler.enabled)
```

Comment:
Values.controlPlane.distro.k8s.enabled is false by default but implicitly enabled if no other distro is enabled